### ISSUE TITLE
DefinitionMethods not included in implementing ObjectType classes

### ISSUE THREAD

**Issue by joostverdoorn** [rmosolgo/graphql-ruby#1625]
Given the following interface definition:
```ruby
module MyInterface
  include GrahpQL::Schema::Interface

  definition_methods do
    def my_special_definition_method
       ...
    end
  end
end
```

I cannot use the following object definition:
```ruby
class MyObject < GraphQL::Schema::Object
  implements MyInterface
  my_special_definition_method
end
```

Throwing the following error:
```
NoMethodError: undefined method `my_special_definition_method' for MyObject:Class
```

Such functionality would be very useful for me, but methods defined within `definition_methods` are currently not included within implementing classes. Is there a particular reason why these methods are not included? Or do you think it's possible to actually include them?

**Comment by rmosolgo** [comment]
Oh, that _would_ be nice! No good reason not to include them, I just didn't think of it. Do you want to take a stab at it? Happy to work together on it if you're interested.

**Comment by joostverdoorn** [comment]
Sure, take a look at https://github.com/rmosolgo/graphql-ruby/pull/1635. I'm probably still missing some things but this seems to get the main job done.

**Comment by rmosolgo** [comment]
Fixed by #1635
